client/components/section-nav/tabs.jsx

```diff
@@ -139,7 +140,7 @@ var NavTabs = React.createClass( {
 				return;
 			}
 
-			navGroupWidth = this.refs.navGroup.getDOMNode().offsetWidth;
+			navGroupWidth = this.refs.navGroup.offsetWidth;
```

Comment:
Here we use `this.refs.navGroup.offesetWidth` and above we use `ReactDom.findDOMNode( this.refs[ 'tab-' + index ] ).offsetWidth`. It seems to me that one of these is incorrect or at least redundant?